### ISSUE TITLE
Error when trying to save Metadata

### ISSUE THREAD

**Issue by gnacoding** [Stability-AI/StableSwarmUI#145]
Hi All

I couldn't find an issue regarding this error.

StableSwarmUI Version 0.5.8.0

A error occurs when I try to do the following:
``` Lora Tab > Click on Burger Menu in a Lora > Edit Metadata > Click Save in new Pop-Up Window ``` 

Console Log:

> [Error] [WebAPI] Error handling API request '/API/EditModelMetadata': Internal exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at StableSwarmUI.WebAPI.T2IAPI.EditModelMetadata(Session session, String model, String title, String author, String type, String description, Int32 standard_width, Int32 standard_height, String preview_image, String usage_hint, String date, String license, String trigger_phrase, String tags, String subtype) in C:\Users\AI\Documents\AI\stableswarmui\5.8\StableSwarmUI\src\WebAPI\T2IAPI.cs:line 514
   at StableSwarmUI.WebAPI.API.HandleAsyncRequest(HttpContext context) in C:\Users\AI\Documents\AI\stableswarmui\5.8\StableSwarmUI\src\WebAPI\API.cs:line 145

Line 514 in T2IAPI is:
actualModel.Metadata.PreviewImage = preview_image;

Seems it has something to do with the preview image
Happy to provide more infos if needed!

EDIT: Additional Info
When I toggle off ` Save Image ` in the Metadata Editor I don't get an error and the Image and Description will be saved. But when I refresh the Lora Library, the changes are gone.

**Comment by mcmonkey4eva** [comment]
Oh, no, that was a minor code bork, fixed^

If saving fails again after updating, let me know.

**Comment by gnacoding** [comment]
> Oh, no, that was a minor code bork, fixed^
> 
> If saving fails again after updating, let me know.

Glad it was only a minor issue! Thanks for your fast help.
I pulled the repository and tested it again. Now the image is added without an error but unfortunately it's not persistent. Once I refresh or restart stableswarm, the Info (and image) is gone from the Metadata. 

I don't see any logs when saving the metadata. Is there a debug argument I can set when launching stableswarm?

**Comment by mcmonkey4eva** [comment]
For debugging, You can go to Server -> Server Configuration and edit `LogLevel` to `Debug`

I realized that if there was an error saving the model after metadata edits it could be swallowed silently, and pushed a commit that makes sure it gets logged.

**Comment by gnacoding** [comment]
> For debugging, You can go to Server -> Server Configuration and edit `LogLevel` to `Debug`
> 
> I realized that if there was an error saving the model after metadata edits it could be swallowed silently, and pushed a commit that makes sure it gets logged.

EDIT:
After writing this below I discovered something:
When I'm in the ` Models ` tab and I use the file browser (in the gui on the far left) to navigate to the Lora folder and edit the metadata there, everything is saved and persists after a refresh. But when I'm in the Lora Tab the changes are still not shown even after a refresh.

I have a custom folder structure where the .safetensors are outside the StableSwarm folder (could this have something to do with it?)
ModelRoot C:\Users\AI\Documents\AI\Models-Stable-Diffusion\
SDModelFolder Model
SDLoraFolder Model/Lora

/EDIT
------------

Thanks for pointing me to the Log-Settings location (totally missed it)
I made two tests:

1. Edit Metadata without generating an image first.
2. Edit Metadata after generating an Image first while using the LoRa I want to edit.

In both cases clicking on ` Edit Metadata ` and ` Save ` doesn't provide any Log in ` Debug ` or even ` Verbose ` .

After editing the Metadata when I hit the refresh button, the following Debug Log is generated:
` [Debug] Model Lora/add-detail-xl.safetensors seems to match type Stable Diffusion XL 1.0-Base LoRA `

after that it resets all entries of the Lora.

Thank you for all your hard work. StableSwarm is the only tool which generates consistent outputs for me (propably due to the ComfyUI backend).
